readme.md

```diff
@@ -5,25 +5,42 @@
 * [Motivation](#motivation)
 * [Repository Structure](#repository-structure)
 * [Contributing](#contributing)
+* [Requirements](#requirements)
+* [How to Develop](#how-to-develop)
+* [How to Test](#how-to-test)
 
 ## Motivation
 
 The purpose of this repository is to open-source general UI components in use across Kiwi.com.
 
-It also serves as a testing ground for the best approach when it comes to CSS-in-JS. We have chosen [Styled components](https://www.styled-components.com) and [Styled JSX](https://github.com/zeit/styled-jsx) as the two front-runners. In time this repository will be changed to use only one technology. This discussion is aggregated here: https://github.com/kiwicom/react-kiwi-ui/issues/7.
+It also serves as a testing ground for the best approach when it comes to CSS-in-JS.
 
 ## Repository Structure
 
-This repository is broken up into 3 projects.
+## Contributing
 
-| Project | Technology                                             | Description                     |
-| :------ | :----------------------------------------------------- | :------------------------------ |
-| BLKN    | [Styled JSX](https://github.com/zeit/styled-jsx)       | All Components for BLKN project |
-| Kiwicom | [Styled Components](https://www.styled-components.com) | All Components for Kiwi.com     |
-| Icons   |                                                        | All Kiwi Icons                  |
+Please checkout our [Contribution Guide](./.github/contributing.md)
 
-The root folder contains the global ESLint rules, Yarn Workspaces and Flow. Please refer to the README files in all project subfolder for information on how to run them.
+## Requirements
 
-## Contributing
+* [Node 8.9+](https://nodejs.org/en/)
+* [Yarn](https://yarnpkg.com/en/)
+
+## How to develop
+
+For local development you can use:
+
+* `yarn storybook`
+
+In case you want to develop in your current project:
+
+* `yarn add @kiwicom/balkan-ui`
+* `yarn watch` in this folder
```

Comment:
I would recommend slightly better notation here, e.g. `in this folder` => `in the root dir of this project`